packages/ui-toolkit/src/card/index.js

```diff
@@ -1,8 +1,8 @@
-export { default } from './card';
-export { default as Outlet } from './outlet';
+export { default as Card } from './card';
```

Comment:
How do you think this can be fixed ?

Reply:
instead of changing the export, you can change the import in index.js: https://github.com/yldio/joyent-portal/compare/kv#diff-0246c19b13126a45967329b01179b6b6L45

Reply:
Instead of changing the export, you can change the import in index.js: https://github.com/yldio/joyent-portal/compare/kv#diff-0246c19b13126a45967329b01179b6b6L45